packages/test/test-service-load/package.json

```
@@ -27,7 +28,12 @@
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
     "start": "node ./dist/nodeStressTest.js",
-    "start:mini": "node ./dist/nodeStressTest.js --profile mini",
+    "start:mini": "node ./dist/nodeStressTest.js --driver tinylicious --profile mini",
```

```suggestion
    "start:mini": "node ./dist/nodeStressTest.js  --profile mini",
```
let's not update the default mode for the mini target